test/unit/specs/service/lineReader.spec.js

```diff
@@ -18,8 +18,9 @@ cleanse kinfolk crucifix marfa gochujang elit chicharrones food truck.`
       const maxLines = 3
       let linesRead = []
       let onCompleteCalled = false
-      const forEachLine = function (line) {
+      const forEachLine = function (line, fileOk) {
```

Comment:
i would suggest using 'isFileOk' in general is suggest using the template is{variableName} for booleans

Reply:
i will change this